### ISSUE TITLE
parameter setting

### ISSUE THREAD

**Issue by Dong-Shiyu** [atomistic-machine-learning/schnetpack#413]
Currently, I use schnetpack to train dipole moment using using QM9 dataset ,but my training always  stop at mae about 0.9 (e angstrom) after about 50 steps.
Here is my parameter setting:
`import schnetpack as spk
import torch
from schnetpack.datasets import QM9

data=QM9("./data/qm9.db",load_only=[QM9.mu],remove_uncharacterized=True)
train,val,test=spk.train_test_split(data=data,num_train=10000,num_val=1000,
split_file="./data/split.npz")
train_loader=spk.AtomsLoader(train,batch_size=100,shuffle=True)
val_loader=spk.AtomsLoader(val,batch_size=100)
# statistic 
mean,stdvar=train_loader.get_statistics(QM9.mu)
schnet=spk.representation.SchNet(n_atom_basis=64,n_filters=64,n_gaussians=200,n_interactions=6,cutoff=20,cutoff_network=spk.nn.cutoff.HardCutoff)
output=spk.atomistic.DipoleMoment(n_in=64,property=QM9.mu,mean=mean[QM9.mu],stddev=stdvar[QM9.mu])
model=spk.AtomisticModel(representation=schnet,output_modules=output)

from torch.optim import Adam
def loss_fn(batch,result):
    diff=batch[QM9.mu]-result[QM9.mu]
    err= torch.mean(diff**2)
    return err
optimizer=Adam(model.parameters(),lr=1e-3)

import schnetpack.train as trn
loss=trn.build_mse_loss([QM9.mu])

metrics=[spk.metrics.MeanAbsoluteError(QM9.mu)]
hooks=[trn.CSVHook(log_path="./log",metrics=metrics),trn.ReduceLROnPlateauHook(optimizer,patience=25,factor=0.5,min_lr=1e-6,stop_after_min=True)]

trainer=trn.Trainer(
    model_path="./model",
    model=model,
    hooks=hooks,
    loss_fn=loss_fn,
    optimizer=optimizer,
    train_loader=train_loader,
    validation_loader=val_loader
)
device="cpu"
n_epochs=200
trainer.train(device=device,n_epochs=n_epochs)
`

I have used a large trainning sets before,but it failed the same way.In order to lower computational cost during parameter searching , I reduce them to 10000,1000 train,val. What is wrong of my parameters?
